rero_ils/modules/circ_policies/jsonschemas/circ_policies/circ_policy-v0.0.1.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "CircPolicy",
+  "description": "JSON schema for circ_policies.",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid",
+    "name",
+    "description",
+    "organisation_pid"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate circ_policies records against.",
+      "type": "string",
+      "minLength": 9
+    },
+    "pid": {
+      "title": "CircPolicy ID",
```

Comment:
Change `CircPolicy` to `Circulation policy`. And check all the JSON schema with this rule.